docs/releases/creating-a-release.md

```diff
@@ -87,10 +87,7 @@ graph LR
   prerelease -.-o|Create Pull Request| main
 ```
 
-After running the action there are two manual steps you must do:
-
-- Create a PR of `release/<release-tag>` into `main` to merge the release branch back into `main`. This PR should be merged after the release is complete.
-- Disable the beta checks on the CI if this is a stable release, or the stable checks if this is a beta relase. This is to ensure that any cherry pick PR's going into release dont run against the wrong build flavor in CI causing failures. Editing the [CI workflow](../../.github/workflows/ci.yml): Modify the output `JSON` string to remove the `beta` flavor for `stable` releases / `stable` flavor for `beta` releases.
+After running the action, you must disable the beta checks on the CI if this is a stable release, or the stable checks if this is a beta relase. This is to ensure that any cherry pick PR's going into release dont run against the wrong build flavor in CI causing failures. Editing the [CI workflow](../../.github/workflows/ci.yml): Modify the output `JSON` string to remove the `beta` flavor for `stable` releases / `stable` flavor for `beta` releases.
```

Comment:
Maybe we can edit this to be about validating that the beta or stable CI is turned off depending on the flavor of release? we are now doing it automatically as part of the workflow so it should just be a go and check then the following is how to do it manually if it didn't work?

Reply:
Good point. Updated.